extensions/default/src/getCustomizationModule.tsx

```diff
@@ -162,6 +165,26 @@ export default function getCustomizationModule({ servicesManager, extensionManag
           id: 'progressDropdownWithServiceComponent',
           component: ProgressDropdownWithService,
         },
+        {
+          id: 'studyBrowser.sortFunctions',
+          merge: 'Append',
+          values: [
+            {
+              label: 'Series Number',
+              sortFunction: (a, b) => {
+                return a?.SeriesNumber - b?.SeriesNumber;
+              },
+            },
+            {
+              label: 'Series Date',
+              sortFunction: (a, b) => {
+                const dateA = new Date(formatDate(a?.SeriesDate));
```

Comment:
If you are going to sort on series date, it HAS to include series time as well - there are cases where doing it wrong can result in patient harm.